src/service/impl/feedServiceImpl.ts

```diff
@@ -0,0 +1,133 @@
+import { Article, Category, Feed, User } from "skiosa-orm";
+import { dataSource } from "skiosa-orm/lib/db";
+import { Arg, FieldResolver, Int, Query, Resolver, Root } from "type-graphql";
+import { PaginationArg } from "../../model/paginationArg";
+import { FeedService } from "../feedService";
+
+@Resolver((_of) => Feed)
+export class FeedResolverService implements FeedService {
```

Comment:
Please rename this class to **FeedServiceImpl** or **FeedResolverImpl**. I would name it Service because the Article Service has the same naming-schema, but we can discuss that if you want.